README.md

```diff
@@ -2,14 +2,14 @@
 
 [![Britecharts Logo][36]][31]
 
-> Britecharts is a client-side **reusable Charting Library** based on [D3.js v4][1] that allows easy and intuitive use of charts and components that can be composed together creating amazing visualizations.
+> Britecharts is a client-side **reusable Charting Library** based on [D3.js v4][1] that allows the easy and intuitive use of charts and components that can be composed together creating amazing visualizations.
```

Comment:
Here, instead of the `that allows the easy and intuitive...`, I would put something like `that offers easy and intuitive...`. The `the easy` part seems awkward.

Reply:
I'd also change "...composed together creating amazing visualizations..." to "...composed together to create amazing visualizations..."

Reply:
Thanks!